styles.css

```diff
@@ -0,0 +1,268 @@
+
+.quote {
+  font-family: 'Swanky and Moo Moo', cursive;
+  font-size: 2em;
+  line-height: 1em;
+  margin: -0.5rem auto -0.5rem auto;
+  padding: 0;
+}
+
+.smalltext{
+  font-size: 1rem;
+}
+
+/* WELCOME SECTION STYLES*/
+
+#welcome-section {
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+  background-color: var(--neon-hot-pink);
+}
+
+.centered-box {
+  margin: 0 auto;
+  padding: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 0;
+  text-align: center;
+}
+
+.welcome-title {
+  font-size: 8rem;
+  color: var(--neon-lime);
+}
+
+/* PROJECTS SECTION STYLES*/
+
+#projects {
+  background-color: var(--neon-orange);
+  margin: 0;
+  padding: 4rem 0 10rem 0;
+}
+
+.centered-title {
+  text-align: center;
+  margin-top: 0;
+}
+
+.projects-grid{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.project-tile {
+  border-bottom: 1px solid rgba(0, 0, 0, 0.7); /* shadows in rgba in case I change the base color later */
+  border-right: 1px solid rgba(0, 0, 0, 0.7);
+  border-left: 2px solid white;
+  border-top: 2px solid white;
+  background-color: var(--window-gray);
+  margin: 1rem auto;
+  width: 30rem;
+  height: 22rem;
+  text-align: center;
+  overflow: hidden;
+  box-shadow: 0.4rem 1rem 0.4rem rgba(0, 0, 0, 0.3);
+}
+
+.project-title {
+  text-align: center;
+  font-family: 'Fira Mono', monospace;
+  color: #FFF;
+  background-color: var(--neon-blue);
+  font-size: 1.4rem;
+  padding: 0.2em;
+  margin: 0;
+  /* highlights and shadows in rgba in case I change the base color later */
+  border-top: 2px solid rgba(255, 255, 255, 0.3);
+  border-left: 2px solid rgba(255, 255, 255, 0.3);
+  border-right: 2px solid rgba(0, 0, 0, 0.3);
+  border-bottom: 2px solid rgba(0, 0, 0, 0.3);
+}
+
+.project-thumb {
+  height: auto;
+  width: 100%;
+  background-color: var(--neon-purple);
+  margin: 0 auto
+  max-width: 100%;
+  border 1px aqua;
+}
+
+/* ABOUT SECTION STYLES*/
+
+#about-contact {
+  padding: 4rem 2rem;
+  margin: 0 auto;
+  background-color: var(--neon-purple);
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  grid-gap: 3rem;
+  grid-template-areas: 'ab-image ab-text' 'ab-image ab-links';
+  color: var(--neon-lime);
+}
+
+#about-picture {
+  width: 100%;
+  max-width: 40rem;
+  grid-area: ab-image;
+  margin: 0 auto;
+}
+
+.bordered-picture {
+  width: inherit;
+  border: 1rem solid #EEE;
+  box-shadow: 1rem 1rem 0.5rem rgba(0, 0, 0, 0.4);
+}
+
+#about-text {
+  grid-area: ab-text;
+}
+
+#about-links {
+  grid-area: ab-links;
+  display: flex;
+  justify-content: space-between;
+  align-content: flex-start;
+}
+
+.profile-link {
+  text-decoration: none;
+  color: #FFF;
+  font-weight: bolder;
+}
+
+/* FOOTER STYLES*/
+
+#site-footer {
+  padding: 0 2rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+#sitemap {
+
+}
+
+.sitemap-links{
+
+}
+
+#copyright {
+  justify-self: center;
+  align-self: center;
+}
+
+#footerlogo{
```

Comment:
Are this empty classes going to have any css rules in them or not? If not, they should be deleted.

Reply:
They will, they will. They are there to remind me to finish them.